### PR TITLE
Bug 1740064 - Increase TCP rollout to 100% including aurora excluding study opt-out

### PR DESCRIPTION
**Pull request by Mardak**

### CODE REVIEW

File: cfr.json [hunk 1]
```diff
@@ -445,6 +445,6 @@
...
+    "targeting": "source == 'startup' && !isMajorUpgrade && 'app.shield.optoutstudies.enabled'|preferenceValue && browserSettings.update.channel in ['release', 'beta', 'aurora'] && version|versionCompare('100.0.1') >= 0 && 'network.cookie.cookieBehavior'|preferenceValue == 4 && !'privacy.restrict3rdpartystorage.rollout.enabledByDefault'|preferenceIsUserSet && !'browser.privatebrowsing.autostart'|preferenceValue && !'browser.search.param.google_channel_us'|preferenceValue('')|regExpMatch('^[ntc]us5$') && !'browser.search.param.google_channel_row'|preferenceValue('')|regExpMatch('^[ntc]row5$') && currentDate|date - profileAgeCreated >= 10800000"
```
Comment (punamdahiya) [inline cfr.json:448]: My understanding rollout don't show up in about:studies. Curious if  'app.shield.optoutstudies.enabled' check valid here as it's not a experiment but rollout.  Will browsing setting of ['do not recommend features/extension' pref ](https://searchfox.org/mozilla-central/rev/301e9545863b14ff85aa110fabd4274da4593f9d/browser/components/enterprisepolicies/Policies.jsm#722)makes more sense.

Comment (punamdahiya) [inline cfr.json:448]: My understanding rollout don't show up in about:studies. Curious if  'app.shield.optoutstudies.enabled' check valid here as it's not a experiment but rollout.  Will browsing setting of ['do not recommend features/extension' pref ](https://searchfox.org/mozilla-central/rev/301e9545863b14ff85aa110fabd4274da4593f9d/browser/components/enterprisepolicies/Policies.jsm#722)makes more sense.

File: cfr.json [hunk 2]
```diff
@@ -397,7 +397,7 @@
           "label": {
             "string_id": "spotlight-total-cookie-protection-header"
           },
-          "size": "20px"
+          "size": "23px"
```
Comment (punamdahiya) [inline cfr.json:400]: is the size change intended?

**Review by punamdahiya — APPROVED** [review]
Pref as finalized by product and profile age check in targeting LGTM!

File: cfr.json (hunk 2 above)
Comment (Mardak) [reply cfr.json:400]: Yeah, I tested on a bunch of locales and zoom now that https://bugzilla.mozilla.org/show_bug.cgi?id=1767606 was uplifted to release 100.0.1. Victoria had concerns that the existing title and subtitle size/spacing were too similar.

File: cfr.json (hunk 1 above)
Comment (Mardak) [reply cfr.json:448]: Yeah, it was requested to follow the same pref as nimbus and not the feature recommendation one.

**Pull request merged**